doc/api/corepack.md

```diff
@@ -0,0 +1,95 @@
+# Corepack
+
+<!-- introduced_in=REPLACEME -->
+<!-- type=misc -->
+
+> Stability: 1 - Experimental
+
+_[Corepack][]_ is an experimental tool shipped with Node.js to help with
+managing versions of your package managers. It exposes binary proxies for
+[Yarn][] and [pnpm][] that, when called, locate the package managers configured
+for your local projects, transparently install them, and finally run them
+without requiring explicit user interaction.
+
+This feature allows you to make sure that everyone in your team is using
+exactly the package manager version you want them to without having to check
+them into your repository or managing complex migrations across developers'
+systems.
+
+## Workflows
+
+### Enabling the feature
+
+Due to its experimental status Corepack currently needs to be explicitly
+enabled to have any effect. To do that simply run [`corepack enable`][], which
+will set up the symlinks in your environment, next to the `node` binary
+(and overwrite the existing symlinks if necessary).
+
+From this point forward, any call to the `yarn`, `pnpm`, or `pnpx` binaries
+will work without further setup. Should you experience a problem, just run
+[`corepack disable`][] to remove the proxies from your system (and consider
+opening up an issue on the [Corepack repository][] to let us know).
+
+### Configuring a package
+
+The Corepack proxies will find the closest [`package.json`][] files in your
```

Comment:
Do you really mean plural here? There should be only one _closest_ `package.json` right?
```suggestion
The Corepack proxies will find the closest [`package.json`][] file in your
```